### DISCUSSION TITLE
Do i need to restart Radicale every time my SSL certificate renews?

### DISCUSSION THREAD

**Discussion by Volper212** [Kozea/Radicale#1360] [Q&A general]
It's set up as a symlink with certbot and is renewed automatically. Will I have to set up something to restart Radicale when that happens?

**Comment by pbiering** [comment]
I assume yes, this is also required for a lot of other daemons which at least requires "reload" or "restart" (depending on internal implementation) to release their current server socket with somehow cached TLS server certificate+key and open a new one.
